claim/schema.py

```diff
@@ -112,16 +117,26 @@ def resolve_claim_attachments(self, info, **kwargs):
         if not info.context.user.has_perms(ClaimConfig.gql_query_claims_perms):
             raise PermissionDenied(_("unauthorized"))
 
-    def resolve_claim_admins(self, info, search=None, **kwargs):
+    def resolve_claim_admins(
+            self,
+            info,
+            search=None,
+            user_health_facility=None,
+            **kwargs
+    ):
         if not info.context.user.has_perms(ClaimConfig.gql_query_claim_admins_perms):
             raise PermissionDenied(_("unauthorized"))
 
+        queryset = ClaimAdmin.objects
+        if user_health_facility is not None:
+            queryset = queryset.filter(health_facility=user_health_facility)
```

Comment:
Shouldn't it be 
```suggestion
            queryset = queryset.filter(health_facility__uuid=user_health_facility)
```
?

Reply:
I'm really confused, it works in both so I will keep your change to make it more readable